packages/devtools_app/lib/src/screens/profiler/cpu_profile_controller.dart

```diff
@@ -507,11 +507,8 @@ class CpuProfilerController
 
   void updateView(CpuProfilerViewType view) {
     _viewType.value = view;
-    _dataNotifier.value = cpuProfileStore
-        .lookupProfile(
-          label: _wrapWithFilterSuffix(userTagNone),
-        )
-        ?.getActive(view);
+    _dataNotifier.value =
+        cpuProfileStore.lookupProfile(label: userTagNone)?.getActive(view);
```

Comment:
why are we no longer including the filter suffix?

Reply:
`loadProcessedData` doesn't seem to include the filter suffix and was causing tests to fail. Should we just add the filter suffix there as well?

Reply:
yes I think we need a filter suffix in both places.

Reply:
well I guess really we need to understand how tests were passing before this PR? and what about this PR changed things so that they fail?

Reply:
Sorry, I wasn't clear. It was causing the new tests to fail since we didn't have a filter suffix in `loadProcessedData`.